### PR TITLE
feat(docs): noindex on staging deployment

### LINKED ISSUE #240: [Needs Expansion]: Put noindex on staging docs
### Idea

Prevent Google indexing of staging pages

### Possible Solutions

_No response_

### Extra Information

_No response_

### PR DESCRIPTION
**Pull request by JoltCode**
# Description

Search engines should no longer index the staging version if a `STAGING = "1"` env variable is present.

Fixes #240 (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# How Has This Been Tested?

N/A

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works 
- [ ] I have updated .env.example if I added a new environment variable
- [x] My PR title follows the [semantic commits](https://www.conventionalcommits.org/en/v1.0.0/) style

### CODE REVIEW

**Comment by JoltCode** [comment]
Sure! 😄

**Comment by JoltCode** [comment]
Ready for review 😄

File: .env.example [hunk 1]
```diff
@@ -24,6 +24,9 @@ ELASTICSEARCH_MESSAGE_INDEX = messages
 REDIS_URL = redis://:redis@ao-cache-redis:6379
 NEXTAUTH_SECRET = 123
 
+# Use this if deploying in a staging enviroment
+# STAGING = "1"
```
Comment (RhysSullivan) [inline .env.example:28]: change from stagining 1 to DEPOYMENT_ENV, options are development, staging, production

**Review by RhysSullivan — CHANGES_REQUESTED** [review]
good start, went over needed changes w/ you on call

**Pull request merged**